rotkehlchen/rotkehlchen.py

```diff
@@ -256,10 +258,20 @@ def unlock_user(
             greenlet_manager=self.greenlet_manager,
             connect_at_start=ETHEREUM_NODES_TO_CONNECT_AT_START,
         )
+
+        # TODO: decide how do we want it in ChainManager:
+        # 1. A single argument/property (i.e. substrate_manager=substrate_manager)
+        # 2. An argument/property per substrate chain (as for now)
+        kusama_manager = SubstrateManager(
+            chain=str(SubstrateChain.KUSAMA),
+            msg_aggregator=self.msg_aggregator,
+        )
+
         Inquirer().inject_ethereum(ethereum_manager)
         self.chain_manager = ChainManager(
             blockchain_accounts=self.data.db.get_blockchain_accounts(),
             ethereum_manager=ethereum_manager,
+            kusama_manager=kusama_manager,  # TODO: reconsider maintainability
```

Comment:
It's fine like this for now. At some point this needs to be refactored but probably todo not required.